homeassistant/components/sensor/miflora.py

```diff
@@ -113,6 +103,14 @@ def __init__(self, poller, parameter, name, unit, force_update, median):
         # Use median_count = 1 if no filtering is required.
         self.median_count = median
 
+    async def async_added_to_hass(self):
+        """Set initial state."""
+        @callback
+        def on_startup():
+            self.async_schedule_update_ha_state(True)
+
+        self.hass.bus.async_listen_once(EVENT_HOMEASSISTANT_START, on_startup)
+
```

Comment:
I think you don't need this code, because it's a polling device and the core call the update itself after some time. And if I understand that correct, time is not a problem only real time :D

Reply:
`after some time` == `scan_interval`? If it is, then waiting 20 minutes to see the state is too much. I need something like `update_before_add=True`, but w/o HA startup block.